.github/workflows/test.yml

```diff
@@ -59,14 +59,7 @@ jobs:
 
       - name: Integration Test
         run: |
-          echo "Testing initial install"
-          ./install.sh
-          ./_integration-test/run.sh
-          echo "Testing in-place upgrade"
-          # Also test plugin installation here
-          echo "sentry-auth-oidc" >> sentry/requirements.txt
-          ./install.sh --minimize-downtime
-          ./_integration-test/run.sh
+          ./test.sh
```

Comment:
Can you please move this change to another PR?

Reply:
I also modified the steps, it was not just a pure refactoring. If you insist I can revert the move, but I think it is more convenient for testing to keep it.

Reply:
We use `test.sh` over here:

https://github.com/getsentry/sentry/blob/fe4795f5eae9e0d7c33e0ecb736c9d1369535eca/docker/cloudbuild.yaml#L59

So let's keep changes to that isolated as it can break on the other side of the world.

Reply:
Especially now that the complexity of steps has grown, I think it is much preferrable to have a dedicated file for it.

Reply:
Got it, I extracted the more involved tests to a separate file and documented the dependency to the other repository.